src/main/java/com/synopsys/integration/detect/configuration/DetectableOptionFactory.java

```diff
@@ -139,7 +141,13 @@ public GradleInspectorOptions createGradleInspectorOptions() {
         final String onlineInspectorVersion = detectConfiguration.getProperty(DetectProperty.DETECT_GRADLE_INSPECTOR_VERSION, PropertyAuthority.NONE);
         final GradleInspectorScriptOptions scriptOptions = new GradleInspectorScriptOptions(excludedProjectNames, includedProjectNames, excludedConfigurationNames, includedConfigurationNames, customRepository, onlineInspectorVersion);
         final String gradleBuildCommand = detectConfiguration.getProperty(DetectProperty.DETECT_GRADLE_BUILD_COMMAND, PropertyAuthority.NONE);
-        return new GradleInspectorOptions(gradleBuildCommand, scriptOptions);
+
+        final String proxyHost = detectConfiguration.getProperty(DetectProperty.BLACKDUCK_PROXY_HOST, PropertyAuthority.NONE);
+        final String proxyPort = detectConfiguration.getProperty(DetectProperty.BLACKDUCK_PROXY_PORT, PropertyAuthority.NONE);
+        final String nonProxyHosts = detectConfiguration.getProperty(DetectProperty.BLACKDUCK_PROXY_IGNORED_HOSTS, PropertyAuthority.NONE);
```

Comment:
What about the other proxy properties? Such as blackduck.proxy.username, blackduck.proxy.password, blackduck.proxy.ntlm.workstation, blackduck.proxy.ntlm.domain

Reply:
I would get the ProxyInfo from ConnectionManager if you can. https://github.com/blackducksoftware/synopsys-detect/blob/adf1481099b9aad68ab34450ed7510e5b3e7c099/detect-configuration/src/main/java/com/synopsys/integration/detect/configuration/ConnectionManager.java#L53